client/my-sites/customer-home/cards/tasks/site-resume-copy/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import siteCopyIllustration from 'calypso/assets/images/customer-home/illustration--import-complete.svg';
+import { useSiteCopy } from 'calypso/landing/stepper/hooks/use-site-copy';
+import { preventWidows } from 'calypso/lib/formatting';
+import { TASK_SITE_RESUME_COPY } from 'calypso/my-sites/customer-home/cards/constants';
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { getSite, getSiteOption } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+
+const SiteResumeCopy = ( { siteSlug, sourceSite, sourceSiteSlug } ) => {
+	const translate = useTranslate();
+	const { startSiteCopy } = useSiteCopy( sourceSite );
+
+	return (
+		<Task
+			isUrgent
+			title={ translate( 'Ready to finish copying your site?' ) }
+			description={ preventWidows(
+				translate(
+					'Your copied site is almost there. You can finish setting it up any time you’re ready.'
+				)
+			) }
+			actionText={ translate( 'Finish setting it up' ) }
+			actionUrl={ `/setup/copy-site?sourceSlug=${ sourceSiteSlug }&siteSlug=${ siteSlug }` }
+			actionOnClick={ () => {
+				startSiteCopy( 'calypso_resume_copy_site_click' );
```

Comment:
FYI - I moved the Tracks call outside of the hook: https://github.com/Automattic/wp-calypso/pull/72291/commits/f0f2a81fc7caa2173fd295bbef75086b8b23d422

Reply:
Thanks, I've rebased on the trunk and updated it according to that commit.

Reply:
@wojtekn Did you register the new Tracks event?

Reply:
@danielbachhuber just now 1394-gh-Automattic/tracks-events-registration